packages/devtools/lib/src/main.dart

```diff
@@ -67,14 +68,23 @@ class PerfToolFramework extends Framework {
           .open('https://github.com/flutter/devtools/issues', '_feedback');
     });
 
+    final settingsScreen = SettingsScreen();
+    addScreen(settingsScreen);
+    queryId('settings-button')
+      ..setAttribute('href', settingsScreen.ref)
+      ..onClick.listen((e) {
+        e.preventDefault();
+        navigateTo(settingsScreen.id);
+      });
+
     await serviceManager.serviceAvailable.future;
     await addScreens();
     screensReady.complete();
 
     final CoreElement mainNav = CoreElement.from(queryId('main-nav'));
     mainNav.clear();
 
-    for (Screen screen in screens) {
+    for (Screen screen in screens.where((screen) => screen.id != 'settings')) {
       final CoreElement link = CoreElement('a')
         ..add(<CoreElement>[
```

Comment:
One thing to watch out is we have code to highlight that the current screen is selected that may need a tweak given your change to still work... or maybe it still works... through the magic of query selectors. :)
```dart
  void updatePage() {
    // nav
    for (Element element in querySelectorAll('#main-nav a')) {
      final CoreElement e = CoreElement.from(element);
      final bool isCurrent = current.ref == element.attributes['href'];
      e.toggleClass('active', isCurrent);
    }
  }
```

Reply:
actually it still works :D
![image](https://user-images.githubusercontent.com/25892469/63054767-b219e680-bf0e-11e9-8a98-9893ade3e483.png)

I added listener in SettingsScreen to rebuild the settings when the state changed from not connected to connected, is it fine?

For now I'm adding TODOs on what to display when there's no connected app yet?